extension/netflix-page-code.js

```diff
@@ -53,25 +60,27 @@ function embeddedCode() {
   }
 
   const onSyncFunction = (player, syncGMTTs, syncVideoTargetTs) => {
-
-    const MAX_DESYNC_DELTA = 3 * MS_IN_SEC;
-
-    // recalculate these
-    const currentGMTTs = Date.now() / MS_IN_SEC + currentTimeToActualGMTOffset;
-    // time between now and when the video should start
-    const timeToVideoStartSec = syncGMTTs - currentGMTTs - syncVideoTargetTs;
-    const timeToVideoStartMs = timeToVideoStartSec * MS_IN_SEC;
-    const targetPlayerTime = -1 * timeToVideoStartMs;
-
-    const currentPlayerTime = player.getCurrentTime();
-    const delta = Math.abs(targetPlayerTime - currentPlayerTime);
-    if (delta && delta > MAX_DESYNC_DELTA) {
-      // resync
-      player.seek(targetPlayerTime);
-      player.play();
-      // alert the viewer if the video has already ended
-      if (player.isEnded()) {
-        alert('The scheduled video has ended');
+    //only sync if video is playing
+    if (getVideoPlayer().isVideoPlayingForSessionId(playerSessionId)) {
```

Comment:
can you add a check to make sure this is not null? idk what kind of whack race conditions we may not have anticipated